### DISCUSSION TITLE
Feature Extraction - new labels using league

### DISCUSSION THREAD

**Discussion by ravi7522** [ravi7522/Football-Prediction#15] [General]
We can add total no of win, loss, draw for both home and away teams.
We can add league wise win, loss, draw for both home and away teams.

**Comment by ravi7522** [comment]
Please provide your thoughts on this

**Comment by zeelsheladiya** [comment]
total win, loose, and draw by match has been done

**Comment by zeelsheladiya** [comment]
do we need this table for league wise win, loss, draw for both home and away teams.?

<img width="670" alt="image" src="https://github.com/ravi7522/Football-Prediction/assets/46935793/98d31a1d-36ce-44e7-b4f9-c880e22ff887">

@ravi7522 @babuaditya97 @sudipbhattarai820

**Reply by zeelsheladiya** [reply]
according to this table league wise operation has been done now just need to approval for the merge

**Comment by ravi7522** [comment]
convert it to team wise not using total win or loss.

**Comment by zeelsheladiya** [comment]
<img width="916" alt="image" src="https://github.com/ravi7522/Football-Prediction/assets/46935793/ee4d00c2-d71d-49ee-a4c2-466665772b5e">

**Reply by zeelsheladiya** [reply]
@ravi7522 what do you will it gonna work?

**Reply by zeelsheladiya** [reply]
here is the table about league result away team wise

<img width="916" alt="image" src="https://github.com/ravi7522/Football-Prediction/assets/46935793/f5090c6e-ae75-4df1-af14-8e92d6b52d48">
